inspector/src/components/actionTabs/tabs/propertyGrids/animations/keyframeSvgPoint.tsx

```diff
@@ -1,59 +1,101 @@
-    selected: boolean;
-    isLeftActive: boolean;
-    isRightActive: boolean;
-    curveId?: ICurveMetaData;
+  keyframePoint: Vector2;
+  rightControlPoint: Vector2 | null;
+  leftControlPoint: Vector2 | null;
+  id: string;
+  selected: boolean;
+  isLeftActive: boolean;
+  isRightActive: boolean;
+  curveId?: ICurveMetaData;
 }
 
 export interface ICurveMetaData {
-    id: number;
-    animationName: string;
-    property: string;
+  id: number;
+  animationName: string;
+  property: string;
 }
 
 interface IKeyframeSvgPointProps {
-    keyframePoint: Vector2;
-    leftControlPoint: Vector2 | null;
-    rightControlPoint: Vector2 | null;
-    id: string;
-    selected: boolean;
-    selectKeyframe: (id: string) => void;
-    selectedControlPoint: (type: string, id: string) => void;
-    isLeftActive: boolean;
-    isRightActive: boolean;
+  keyframePoint: Vector2;
+  leftControlPoint: Vector2 | null;
+  rightControlPoint: Vector2 | null;
+  id: string;
+  selected: boolean;
+  selectKeyframe: (id: string, multiselect: boolean) => void;
+  selectedControlPoint: (type: string, id: string) => void;
+  isLeftActive: boolean;
+  isRightActive: boolean;
 }
 
-export class KeyframeSvgPoint extends React.Component<IKeyframeSvgPointProps>{
+export class KeyframeSvgPoint extends React.Component<IKeyframeSvgPointProps> {
+  constructor(props: IKeyframeSvgPointProps) {
+    super(props);
+  }
 
-    constructor(props: IKeyframeSvgPointProps) {
-        super(props);
+  select(e: React.MouseEvent<SVGImageElement>) {
+    e.preventDefault();
+    let multiSelect = false;
+    if (e.buttons === 0 && e.ctrlKey) {
+      multiSelect = true;
     }
+    this.props.selectKeyframe(this.props.id, multiSelect);
+  }
 
-    select() {
-        this.props.selectKeyframe(this.props.id);
-    }
-
-    render() {
-        return (
-            <>
-                <svg className="draggable" x={this.props.keyframePoint.x} y={this.props.keyframePoint.y} style={{ overflow: 'visible', cursor: 'pointer' }} >
-                    <image data-id={this.props.id} className="draggable" x="-1" y="-1.5" width="3" height="3" href={this.props.selected ? keySelected : keyInactive} onClick={() => this.select()} />
-                </svg>
-                {this.props.leftControlPoint && <AnchorSvgPoint type="left" index={this.props.id} control={this.props.leftControlPoint} anchor={this.props.keyframePoint} active={this.props.selected} selected={this.props.isLeftActive} selectControlPoint={(type: string) => this.props.selectedControlPoint(type, this.props.id)} />}
-                {this.props.rightControlPoint && <AnchorSvgPoint type="right" index={this.props.id} control={this.props.rightControlPoint} anchor={this.props.keyframePoint} active={this.props.selected} selected={this.props.isRightActive} selectControlPoint={(type: string) => this.props.selectedControlPoint(type, this.props.id)} />}
-            </>
-        )
-    }
-} 
\ No newline at end of file
+  render() {

```

Comment:
is that a formatting-only change? What formatter-configuration are you using? Feels like it will be reverted next time someone else formats the code :-)

Reply:
I'm using prettier.... let me check my config.

Reply:
Most of the team uses the default formatter for vscode, but prettier is perfectly fine. As we don't have any editor/formatter configuration for this project, there is little you can do (sorry, just checked). If you can try using the vscode formatter for this PR, please do. If not, it's perfectly acceptable as it is.